cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -1,27 +1,29 @@
+{% set version = "0.1.0-alpha1" -%}
+
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: kube-ui-v4
+  name: kubernetes-dashboard-{{ version }}
   namespace: kube-system
   labels:
-    k8s-app: kube-ui
-    version: v4
+    k8s-app: kubernetes-dashboard
+    version: {{ version }}
     kubernetes.io/cluster-service: "true"
 spec:
   replicas: 1
   selector:
-    k8s-app: kube-ui
-    version: v4
+    k8s-app: kubernetes-dashboard
+    version: {{ version }}
   template:
     metadata:
       labels:
-        k8s-app: kube-ui
-        version: v4
+        k8s-app: kubernetes-dashboard
+        version: {{ version }}
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
-      - name: kube-ui
-        image: gcr.io/google_containers/kube-ui:v4
+      - name: kubernetes-dashboard
+        image: gcr.io/google_containers/kubernetes-dashboard:{{ version }}
```

Comment:
Then with cross-compiling I assume images will be named `gcr.io/google_containers/kubernetes-dashboard-{{arch}}:{{version}}` with arch defaulting to `amd64`
WDYT?
It would be great to name the images the right way from the beginning